tests/unit/snow/helpers/functional_test_cases.py

```diff
@@ -0,0 +1,56 @@
+import sqlglot
+from sqlglot import ParseError, UnsupportedError
+
+
+class FunctionalTestFile:
+    """A single test case with the required options"""
+
+    def __init__(self, databricks_sql: str, source: str, test_name: str):
+        self.databricks_sql = databricks_sql
+        self.source = source
+        self.test_name = test_name
+
+
+class FunctionalTestFileWithExpectedException:
+    """A single test case with the required options and expected exceptions"""
+
+    def __init__(self, databricks_sql: str, source: str, test_name: str, expected_exception: sqlglot.errors):
```

Comment:
expected_exception should have type sqlglot.errors.SqlglotError. sqlglot.errors is a module.

Reply:
Updated the class.